### ISSUE TITLE
How can I limit the number of CPUs to 10 or less during training?

### ISSUE THREAD

**Issue by TheMakiran** [qubvel/segmentation_models.pytorch#499]
I adapted the CamVid example to train on my own dataset. I am running on a shared cluster with 1 GPU and 10  cpus available. I get the following error from the public job scheduler ('pbs'). 

=>> PBS: job killed: ncpus 11.03 exceeded limit 10 (sum)

Is there a way to limit the number of CPUs that pytorch uses for training?

I added the following to the python file:
torch.set_num_threads(8)

and set 'num_workers=10'

but the error persists.
